protocol/messenger_config.go

```diff
@@ -72,6 +73,11 @@ type MessengerSignalsHandler interface {
 	SendCuratedCommunitiesUpdate(response *communities.KnownCommunitiesResponse)
 }
 
+type Manager interface {
```

Comment:
```suggestion
type AccountManager interface {
```

Reply:
It's handled in this PR https://github.com/status-im/status-go/pull/6687